salt/kube-apiserver/apiserver.jinja

```diff
@@ -50,10 +50,13 @@ KUBE_API_ARGS="--advertise-address={{ salt.caasp_net.get_primary_ip() }} \
                --tls-private-key-file={{ pillar['ssl']['kube_apiserver_key'] }} \
                --cert-dir=/etc/pki \
                {{ salt.caasp_pillar.get('components:apiserver:args') }} \
+               --requestheader-allowed-names=aggregator \
```

Comment:
We can't use this flag like this otherwise only a client with the name "aggregator" will be allowed.

Reply:
In my tests, not using this flags was ok with a generated certificate.